BlackLagerMessage.proto

```diff
@@ -1,5 +1,47 @@
 syntax = "proto3";
 
+/*
+ * https://grpc.io/docs/protoc-installation/
+ * Useful protc installation guide for reference
+ */
+
+/*
+ * Can contain signed message or Persona protobuf
+ */
+message BlackLagerMessage{
+
+  /*
+  * Used to differentiate text or persona
+  */
+  int32 type = 1;
+
+  optional BlackLagerText text = 2;
+
+  optional Persona Persona = 3;
+}
+
+/*
+ * Used to store all user owned personas and known peer personas
+ */
+ message Wallet {
+
+  Persona my_personas = 1;
+
+  repeated Persona peer_personas = 2;
+}
+
+/*
+ * Used to store signed message along with sender and receiver info
+ */
+message BlackLagerText{
+
+  bytes signedMessage = 1;
+
+  int32 senderNodeNum = 2;
```

Comment:
Do we need the senderNodeNum and receiverNodeNum? The person receiving the message already knows their nodeNum. Also, messages are broadcast so they are sent to everyone. The receiver also already has the nodeNum of who sent them the message from the Mesh packet.